static/styles/app.css

```diff
@@ -233,3 +233,24 @@ a {
 #nprogress .spinner {
   display: none;
 }
+
+@media (max-width: 767px) {
```

Comment:
This is not the proper way to fix this, there should not be any visual changes for this fix.